api/src/pcapi/core/offers/api.py

```diff
@@ -292,6 +292,8 @@ def update_offer(
                 changed_bookingContact,
                 offer.lastProvider,
             )
+    if description is not UNCHANGED:
+        offer.description = description
```

Comment:
[question] La modification du champ `description` n'est pas déjà prise en charge par la boucle itérant sur `modifications`, plus bas ?